site/en/blog/chromium-chronicle-25/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "The Chromium Chronicle #25: Thread Safety Annotations"
+description: >
+  Learn how Clang's static analysis framework takes over the toil of thread-safety proofs.
+layout: 'layouts/blog-post.njk'
+date: 2021-10-01
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/hgu6uTktp2ipmuODZZhP.jpg'
+alt: >
+  Chromium Chronicle image
+tags:
+  - chromium-chronicle
+---
+
+**Episode 25:** by Victor Costan in SFO (September, 2021)<br>
```

Comment:
Thanks for making all the edits, Rachel! Could we please update this time frame "(September, 2021)" to instead be "(October, 2021)", since we didn't get to push last week?

Reply:
```suggestion
**Episode 25:** by Victor Costan in SFO (October, 2021)<br>
```